File: Math/Palindrome Number/PalindromeNumber.java
class Solution {
    public boolean isPalindrome(int x) {
        int temp = x, sum = 0;
        while (temp > 0) {
            int digit = temp % 10;
            sum = (sum * 10) + digit;
            temp = temp / 10;
        }

        if (sum == x)
            return true;

        return false;
    }
}

File: Math/Palindrome Number/PalindromeNumber.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        if x < 0:
            return False

        rev = 0
        temp = x
        while temp > 0:
            rev = (rev * 10) + (temp % 10)
            temp = temp // 10
        return rev == x
